### DISCUSSION TITLE
Is it possible to edit the default UI of Asp.NET Identity

### DISCUSSION THREAD

**Discussion by AhmedAbuelnour** [dotnet/aspnetcore#27103] [General] [answered]
I'm using Blazor WebAssembly, and used the default Individual User Authentication.

Can i change the default Login/Register Pages?

**Comment by blowdart** [comment, marked as answer]
Yes, via [Scaffolding](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-3.1&tabs=visual-studio).
